src/rebuild/srv.c

```diff
@@ -1295,6 +1296,7 @@ rebuild_task_ult(void *arg)
 		 * rebuild sequence, which has to be done by failure
 		 * sequence order.
 		 */
+		dss_sleep(1000);
```

Comment:
Should this only for specify error?

Reply:
I think if we are sure next retry will work, it's okay to retry without backoff; otherwise, some sort of backoff is required.

This dss_sleep(1s) is a quick hack for reducing the flood of RAS events generated by the rebuild retry here. Some better implementation is welcome. E.g., use sched_req APIs, use d_backoff_seq. Up to you.

Reply:
I am fine with this  for the moment.